content/troubleshooting/preview-built-problem.md

```diff
@@ -69,6 +70,82 @@ runs config file commands from both `update` and `build`. You need to manually
 [Rebuild a Base Preview](/building-a-preview/work-with-base-previews/change-or-update/#change-a-base-preview) to pull a
 new Docker image. {{% /notice %}}
 
+## Using Base Previews - unexpected Preview build results
+
+When you're using Base Previews, you may experience a few different types of unexpected build results:
+
+- [A Preview that I didn't expect to see got built](#a-preview-that-i-didn-t-expect-to-see-got-built)
+- [A Preview that I did expect to see didn't build](#a-preview-that-i-did-expect-to-see-didn-t-build)
```

Comment:
Thought:

- A Preview was built that I didn't expect
- A Preview I expected to build is missing

Reply:
Thanks for the suggestions, @q0rban ! I did think the wording here was awkward, but it was the last thing I was trying to finish up and my brain couldn't come up with anything better. This is an improvement, so I've borrowed and tweaked a little. Thank you!